.travis.yml

```diff
@@ -7,7 +7,7 @@ matrix:
   - env: SBT_VERSION="0.13.18"
     scala: 2.10.7
 
-  - env: SBT_VERSION="1.2.8"
+  - env: SBT_VERSION="1.3.0"
```

Comment:
Let's keep sbt 1.2.8 in the matrix and also add 1.3.0. I'd like to make sure all the tests pass for both 1.2.8 (which uses Ivy) and 1.3.0 (which uses Coursier).